EKF/ekf_helper.cpp

```diff
@@ -1513,6 +1510,47 @@ void Ekf::updateBaroHgtOffset()
 	}
 }
 
+Vector3f Ekf::getVisionVelocityInEkfFrame()
+{
+	// correct velocity for offset relative to IMU
+	const Vector3f pos_offset_body = _params.ev_pos_body - _params.imu_pos_body;
+	const Vector3f vel_offset_body = _ang_rate_delayed_raw % pos_offset_body;
+
+	// rotate measurement into correct earth frame if required
+	if (_ev_sample_delayed.vel_frame == BODY_FRAME_FRD)
```

Comment:
Instead of if/else, could you rather use a `switch` on the enum for the different cases. This way if more get added in the future, it will get compilation errors until the extra cases are added, instead of silently failing.

Reply:
Done